packages/components/src/components/flipcard/flipcard.ts

```diff
@@ -67,10 +70,34 @@ export default class SdFlipcard extends SolidElement {
 
   @state() activeSide: 'front' | 'back' = 'front';
 
+  private focusableElements: HTMLElement[];
+
   connectedCallback() {
     super.connectedCallback();
   }
 
+  firstUpdated() {
+    this.focusableElements = this.getBackFocusableElements();
+  }
+
+  private getBackFocusableElements() {
+    const slot = this.shadowRoot?.querySelector('slot[name="back"]');
+    if (slot instanceof HTMLSlotElement) {
+      const assignedNodes = slot.assignedNodes({ flatten: true });
+      const focusableElements: HTMLElement[] = [];
+
+      assignedNodes.forEach(node => {
+        if (node instanceof HTMLElement) {
+          focusableElements.push(
+            ...node.querySelectorAll<HTMLElement>('a, button, [href], [tabindex]:not([tabindex="-1"])')
```

Comment:
Does this work with sd-button etc. as well?

Reply:
Yes, it does. But I sent a message to Manuel regarding if we actually should do this instead of focusing on the trigger. What do you think?